current-bench.opam

```diff
@@ -66,9 +66,11 @@ build: [
 ]
 dev-repo: "git+https://github.com/ocurrent/current-bench.git"
 pin-depends: [
+  # When modifying the pins in this file, don't forget to also modify pipeline/pipeline.opam
   # Using the changes in ocurrent/ocurrent/pull/421, remove the pin on next release (> 0.6.4)
-  # When modifying this pin, don't forget to also modify pipeline/pipeline.opam
   ["current_github.0.6.4" "git+https://github.com/ocurrent/ocurrent.git#32561e1f2b9cc387d098e0ee918cd6bd2ccd1d3b"]
+  # Using the changes in ocurrent/ocurrent/pull/442, remove the pin on next release (> 0.6.4)
+  ["current_git.0.6.4"    "git+https://github.com/ElectreAAS/ocurrent.git#21e7007e2897d44535e2729b42c80dc1280c6fb1"]
```

Comment:
Should the pins point to ocurrent rather than your repo? (if the changes have been merged)

Reply:
Indeed, good catch